### DISCUSSION TITLE
Beta installation

### DISCUSSION THREAD

**Discussion by roaldarbol** [mooch443/trex#223] [Questions] [answered]
In the Windows issue (#216), you mentioned that it's possible to install the beta version with:
```bash
# windows or linux
conda create -n betarex --override-channels -c trex-beta -c pytorch -c nvidia -c defaults trex
```
That works great it seems (haven't tested the tracking though). The new docs on the otehr hand suggest that we need subsequent `pip` installations - is that still the case? (I get an Access Denied error ,so I'm crossing fingers that it won't be needed) ;-)

**Comment by mooch443** [comment, marked as answer]
No, there are no further commands required.

**Reply by roaldarbol** [reply]
Awesome, thanks!
